scripts/release.kts

```diff
@@ -49,29 +49,42 @@ File(path)
         .forEach { folder ->
             val path = folder.path
             val module = folder.name
+            val pomFile = "$path/pom-default.xml"
 
             val file: String
 
             val androidFile = folder
                     .listFiles { it -> it.name.contains("release") && it.extension == "aar" }
                     .firstOrNull()
 
-            if (androidFile != null) {
-                file = androidFile.path
+            val bomFile = folder
+                .listFiles { it -> it.extension == "jar" }
+                .isEmpty()
+
+            if (bomFile) {
+                val command = "./mvnw gpg:sign-and-deploy-file " +
+                    "-Dfile=$pomFile " +
+                    "-DpomFile=$pomFile " +
+                    "-DrepositoryId=$repositoryId " +
+                    "-Durl=$repositoryUrl " +
+                    "--settings $settingsPath"
```

Comment:
@iker-barriocanal how craft will handle this, as it's a different way of calling maven CLI for a bom file?